src/OmniSharp/Api/Server/OmnisharpController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNet.Mvc;
+using Microsoft.Framework.Runtime;
+
+namespace OmniSharp.Api.Server
+{
+    public partial class OmnisharpController : Controller
```

Comment:
This should be a new controller as it has different dependencies to OmniSharpController. Should be
```public class StopServerController```